src/components/Inputs/inputs.js

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from 'react';
+
+const monthList = [
+  {id: "1", month:"January"},
+  {id: "2", month:"February"},
+  {id: "3", month:"March"},
+  {id: "4", month:"April"},
+  {id: "5", month:"May"},
+  {id: "6", month:"June"},
+  {id: "7", month:"July"},
+  {id: "8", month:"August"},
+  {id: "9", month:"September"},
+  {id: "10", month:"October"},
+  {id: "11", month:"November"},
+  {id: "12", month:"December"}
+];
+
+const ownersList = [{id: "1", name:"Alex"},{id: "2", name:"Luis"}];
+const yearList = [{id: "1", year:"2019"},{id: "2", year:"2018"}];
+
```

Comment:
Also, I think we could move this to a `mockData` file and import them If needed :)

Reply:
I think the "ownersList" I can send this data from the backend, what do you think? and mock the  years data or use MomentJS for the years